tdrs-frontend/src/components/Reports/Reports.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import Button from '../Button'
+import { history } from '../../configureStore'
+import { setYear } from '../../actions/reports'
+
+/**
+ * @param {string} selectedYear = The year that the user has chosen from the
+ * Select component.
+ *
+ * Reports is the home page for users to file a report.
+ * The user can select a year
+ * for the report that they would like to upload and then click on
+ * `Begin Report` to begin uploading files for that year.
+ */
+function Reports() {
+  const selectedYear = useSelector((state) => state.reports.year)
+  const dispatch = useDispatch()
+
+  const handleClick = () => {
```

Comment:
`onClick` of the `Begin Report` button the user will be navigated to a route that includes the year the user selected from the dropdown.